packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -299,3 +302,93 @@ class UncontrolledNavPage extends React.Component {
   }
 }
 ```
+
+### Group section
+
+```js
+import React from 'react';
+import {
+  Page,
+  PageHeader,
+  PageHeaderTools,
+  PageSidebar,
+  PageSection,
+  PageGroup,
```

Comment:
Add the new components to the props at the top of the examples file so they show up in the docs